testprojects/src/go/pants_test/foo.go

```diff
@@ -2,8 +2,8 @@ package main
 
 import (
 	"fmt"
-	"os"
 	"github.com/pantsbuild/pants/testprojects/src/go/pants_test/bar"
+	"os"
```

Comment:
I generally would put a newline between stdlib imports and other imports. Then `gofmt` will sort them as separate blocks.